### DISCUSSION TITLE
AOI with nginx as reverse proxy fine, but errors logged

### DISCUSSION THREAD

**Discussion by SprickW** [nextcloud/all-in-one#5444] [Questions] [answered]
### Steps to reproduce
AIO with nginx.

### Expected behavior 
Logs not filling up with error messages that at least I'm not able to understand

### Actual behavior 

Lots of log entries like those in attached file

#### Host OS 

Proxmox LXC with Ubuntu 22.04 LTS

#### Output of `sudo docker info`

Client: Docker Engine - Community
 Version:    27.3.1
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.17.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.29.7
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 12
  Running: 12
  Paused: 0
  Stopped: 0
 Images: 13
 Server Version: 27.3.1
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: true
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7f7fdf5fed64eb6a7caf99b3e12efcf9d60e311c
 runc version: v1.1.14-0-g2c9f560
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.8.12-2-pve
 Operating System: Ubuntu 22.04.5 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 4GiB
 Name: nextcloud9vcd
 ID: 72066aa7-d91e-45bf-9f79-863105b5b059
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

#### Docker run command or docker-compose file that you used

sudo docker run \
--init \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
--publish 8080:8080 \
--env APACHE_PORT=11000 \
--env APACHE_IP_BINDING=0.0.0.0 \
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
--env NEXTCLOUD_DATADIR="/daten/ncdata" \
nextcloud/all-in-one:latest

#### Other valuable info 

Unable to attach file, sorry

{"reqId":"2SV4bSilOMMPoUPVKAB0","level":3,"time":"2024-10-19T15:10:17+00:00","remoteAddr":"192.168.178.2","user":"--","app":"PHP","method":"GET","url":"/remote.php/dav/","message":"require_once(/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php): Failed to open stream: No such file or directory at /var/www/html/custom_apps/rolls/lib/AppInfo/Application.php#21","userAgent":"Nextcloud Server Crawler","version":"29.0.7.1","data":{"app":"PHP"}}
{"reqId":"2SV4bSilOMMPoUPVKAB0","level":4,"time":"2024-10-19T15:10:17+00:00","remoteAddr":"192.168.178.2","user":"--","app":"rolls","method":"GET","url":"/remote.php/dav/","message":"Error during app service registration: Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')","userAgent":"Nextcloud Server Crawler","version":"29.0.7.1","exception":{"Exception":"Error","Message":"Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')","Code":0,"Trace":[{"file":"/var/www/html/lib/private/AppFramework/Bootstrap/Coordinator.php","line":142,"function":"register","class":"OCA\\Rolls\\AppInfo\\Application","type":"->","args":[["OCP\\AppFramework\\Bootstrap\\IRegistrationContext@anonymous\u0000/var/www/html/lib/private/AppFramework/Bootstrap/RegistrationContext.php:171$5f"]]},{"file":"/var/www/html/lib/private/AppFramework/Bootstrap/Coordinator.php","line":90,"function":"registerApps","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":[["files","activity","admin_audit","analytics","announcementcenter","And 103 more entries, set log level to debug to see all entries"]]},{"file":"/var/www/html/lib/base.php","line":714,"function":"runInitialRegistration","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":[]},{"file":"/var/www/html/lib/base.php","line":1189,"function":"init","class":"OC","type":"::","args":[]},{"file":"/var/www/html/remote.php","line":119,"args":["/var/www/html/lib/base.php"],"function":"require_once"}],"File":"/var/www/html/custom_apps/rolls/lib/AppInfo/Application.php","Line":21,"message":"Error during app service registration: Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')","exception":{},"CustomMessage":"Error during app service registration: Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')"}}
{"reqId":"5VoQ8i1t5FomhduLh3vJ","level":3,"time":"2024-10-19T15:10:34+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"require_once(/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php): Failed to open stream: No such file or directory at /var/www/html/custom_apps/rolls/lib/AppInfo/Application.php#21","userAgent":"--","version":"29.0.7.1","data":{"app":"PHP"}}
{"reqId":"5VoQ8i1t5FomhduLh3vJ","level":4,"time":"2024-10-19T15:10:34+00:00","remoteAddr":"","user":"--","app":"rolls","method":"","url":"--","message":"Error during app service registration: Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')","userAgent":"--","version":"29.0.7.1","exception":{"Exception":"Error","Message":"Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')","Code":0,"Trace":[{"file":"/var/www/html/lib/private/AppFramework/Bootstrap/Coordinator.php","line":142,"function":"register","class":"OCA\\Rolls\\AppInfo\\Application","type":"->","args":[["OCP\\AppFramework\\Bootstrap\\IRegistrationContext@anonymous\u0000/var/www/html/lib/private/AppFramework/Bootstrap/RegistrationContext.php:171$5f"]]},{"file":"/var/www/html/lib/private/AppFramework/Bootstrap/Coordinator.php","line":90,"function":"registerApps","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":[["files","activity","admin_audit","analytics","announcementcenter","And 103 more entries, set log level to debug to see all entries"]]},{"file":"/var/www/html/lib/base.php","line":714,"function":"runInitialRegistration","class":"OC\\AppFramework\\Bootstrap\\Coordinator","type":"->","args":[]},{"file":"/var/www/html/lib/base.php","line":1189,"function":"init","class":"OC","type":"::","args":[]},{"file":"/var/www/html/cron.php","line":58,"args":["/var/www/html/lib/base.php"],"function":"require_once"}],"File":"/var/www/html/custom_apps/rolls/lib/AppInfo/Application.php","Line":21,"message":"Error during app service registration: Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')","exception":{},"CustomMessage":"Error during app service registration: Failed opening required '/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php' (include_path='/var/www/html/3rdparty/pear/archive_tar:/var/www/html/3rdparty/pear/console_getopt:/var/www/html/3rdparty/pear/pear-core-minimal/src:/var/www/html/3rdparty/pear/pear_exception:/var/www/html/apps:/var/www/html/custom_apps')"}}

**Comment by szaimen** [comment, marked as answer]
> require_once(/var/www/html/custom_apps/rolls/lib/AppInfo/../../vendor/autoload.php)

Hi, this seems to be an issue of the rolls app. This needs to be reported to https://github.com/mijorus/rolls/issues
